### DISCUSSION TITLE
1.3 release date

### DISCUSSION THREAD

**Discussion by anatoly71** [probelabs/goreplay#980] [General]
Hello, first of all thanks for a very useful tool and great support.

I wonder, if there is a planned date for the final release of 1.3?

I came across this issue [cannot use http-rewrite-header in version 1.1.0](https://github.com/buger/goreplay/issues/965), then I tried v1.3.0_RC6 but [it crashed on linux](https://github.com/buger/goreplay/issues/969). Then I tried v1.3.0_RC5 and it could rewrite the header correctly but it (as well as v1.3.0_RC6) [does not seem to be able to forward requests to an HTTPS endpoint](https://github.com/buger/goreplay/issues/978).

I really look forward to the release of 1.3, and I wonder if there is a time plan.

**Comment by buger** [comment]
Was released 6 days ago. I guess it is solved now (and your issue with duplicate requests too 🙈 )
